### DISCUSSION TITLE
integration configuration?

### DISCUSSION THREAD

**Discussion by Aham330** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#377] [Q&A]
Hi All,

This looks like an awesome integration for my roborock s7. But I have a real hard time configuring it. I only see the zone and pin to go options. While I have used the map extractor that did extract the maps. could someone maybe copy paste an example config so that the complete functionality of the intergration will become visible?

**Comment by PiotrMachowski** [comment]
Example config is available [here](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card#configuration)

Room cleaning config generator is available [here](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/discussions/317)

**Comment by Aham330** [comment]
Hi Piotr,
Wow thanks for your immediate response. 
I did try around with some of the settings on the configuration examples. But none showed the other selectable icons. As shown in the example card.

![D200C82A-FAA8-4814-A686-5B621E7AE74E](https://user-images.githubusercontent.com/73581209/162489616-d09100ea-f518-47e4-92d2-8912841f9705.png)
